src/FlowerList.js

```diff
@@ -1,11 +1,12 @@
+import Product from "MainContent/Product";
 import React from "react";
 
 function FlowerList() {
   const flowersArray = [...Array(50)];
   return (
     <div className="flower-list">
       {flowersArray.map((item, index) => (
-        <img src="flowersIcon.jpg" />
+        <Product imgSrc="flowersIcon.jpg" title="Palergonija"></Product>
```

Comment:
each child in a list should have a unique "key" prop  - buvau pakūrus šitam bugą nes pamačiau kad meta errorą consolėj https://vuprojects.atlassian.net/browse/GEP-54

Reply:
parasiau komentara apie tai atlassian